Userland/Libraries/LibVT/Terminal.cpp

```diff
@@ -1378,6 +1378,15 @@ void Terminal::handle_key_press(KeyCode key, u32 code_point, u8 flags)
     case KeyCode::Key_PageDown:
         emit_tilde_with_modifier(6);
         return;
+    case KeyCode::Key_Backspace:
+        if (ctrl) {
+            // ANSI delete is within the valid range for CSI codes in Editor.cpp, and
+            // has the same behavior as backspace when emitted with no CSI. This allows
+            // Editor.cpp to delete whole words when Ctrl+Backspace is pressed.
+            emit_final_with_modifier(127);
```

Comment:
Can we add a note here saying that this is an extension?

Reply:
Done, added a little bit more information to the comment as well in case it's ever helpful.